style.css

```diff
@@ -4,6 +4,43 @@
   width: 100vw;
 }
 
+.active {
```

Comment:
## In this Thread I'll try to answer your question and clarify the changes requested by the previous reviewer 👍🏼 

[Screencast from 18 مارس, 2023 CET 11:16:58 م.webm](https://user-images.githubusercontent.com/68030297/226143697-d12a9a60-0105-445d-9085-79e359199ff9.webm)

- As you notice from the video recording, the section with the class `partners` is overlapping on all other sections, this is due to the overuse of the property `position`, the normal overflow that every browser applies to the HTML element is broken.

- [ ] To solve this I highly suggest the following: 

1. Avoid using the `position` property whether it's `relative` or `absolute`, and to fix the layout use properties like `flex`, `grid`, `margin`, and `padding`.

If you need more information about the use of [CSS position](https://www.youtube.com/watch?v=P6UgYq3J3Qs) this tutorial will give a notion of that.

Reply:
Hey 👋🏽 as mentioned by the previous reviewer, some overlapping is happening here, keep going 💪🏽 as this is the last thing preventing your approval, the `Main Program` section is overlapping the header. 👇🏽 

![Screenshot 2023-03-18 at 21 24 51](https://user-images.githubusercontent.com/45739963/226148767-e043ba49-08ba-47a7-9340-ae02c761e525.png)

And the `Partner` section is overlapping the `Speakers`, the `Speakers` is also overlapping the `Navbar` a bit it seems 👀 

![Screenshot 2023-03-18 at 21 29 01](https://user-images.githubusercontent.com/45739963/226148863-768d8159-baaf-4bd4-a4ad-fba8edaee65b.png)

As mentioned by the previous reviewer, avoid using `position: absolute;` as it messes with the positioning and it makes it very hard to control the structure of the page. 🤒

Reply:
Hi! I am having a little issue here, because I don't know how to debug it. The web is looking different on my screen as you can see on this short video, how should I proceed?

https://www.loom.com/share/1698f4a7de804a059ee8c813532cb05a